letsencrypt_service

```diff
@@ -36,14 +37,14 @@ update_certs() {
                 # Symlink to created certificate and key.
                 ln -sf ./$domain/fullchain.pem /etc/nginx/certs/$domain".crt"
                 ln -sf ./$domain/key.pem       /etc/nginx/certs/$domain".key"
-
-                # Update certificates configuration in nginx
-                /usr/local/bin/docker-gen -only-exposed /app/nginx.tmpl /etc/nginx/conf.d/default.conf
-                nginx -s reload
+                reload_nginx='true'
             fi
         done
     done
     unset LETSENCRYPT_CONTAINERS
+    if [[ "$reload_nginx" == 'true' ]]; then
+        /usr/local/bin/docker-gen -only-exposed -notify "nginx -s reload" /app/nginx.tmpl /etc/nginx/conf.d/default.conf
```

Comment:
Sometimes, the configuration will not change, but the certificate will change. Doesn't nginx still need to be reloaded in those cases?